music21/meter/base.py

```diff
@@ -489,6 +541,16 @@ def __init__(self, value: str = '4/4', divisions=None, **keywords):
     def _reprInternal(self):
         return self.ratioString
 
+    def __eq__(self, other) -> bool:
+        '''
+        See notes at class doc and in meter.tests
+        '''
+
+        if not super().__eq__(other):
+            return False
```

Comment:
I think Myke mentioned some inconsistencies exist in music21 with this, but I'd like to move toward `NotImplemented` rather than False, which is what Python defaults to: https://docs.python.org/3/reference/datamodel.html#object.__eq__

Also, I think I first learned of this from mypy, so I'm wondering why mypy doesn't complain (maybe it's an info-level issue)?

Reply:
I'm settling on False, because the base assumption in the new `__eq__` system is that two objects have to be the same class to be equal, so there's no point in returning NotImplemented since it should never find a True value later on.

For `__lt__` or `__gt__` there is still a good reason to return NotImplemented.

Note that bool(NotImplemented) returns True with a deprecation warning for Py3.9 and newer, so including it in a hierarchy of `__eq__` runs into major problems.

I'm doing a major rewrite of MeterSequence/MeterTerminal now, but it's getting a bit out of control.  So maybe we should set up something along the way, but definitely I think slow 6/8 and fast 6/8 should be not equal, same was 4/4 and 2/2.

Reply:
This `__eq__` routine isn't necessary.  The `equalityAttributes` are enough to do the trick.  So delete this routine.

Reply:
Makes sense about the new __eq__ system!

Reply:
Thanks. True. The equalityAttributes are enough here. Deleted.
Further notes on other parts of this discussion below for ease of reference.